src/Reflex/Host/Headless.hs

```diff
@@ -0,0 +1,85 @@
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE ConstraintKinds #-}
+
+module Reflex.Host.Headless where
+
+import Data.Maybe
+import Reflex
+import Control.Monad.Fix
+import Control.Monad.Primitive
+import Reflex.Host.Class
+import Control.Monad.IO.Class
+import Control.Monad.Ref
+import Data.IORef
+import Data.Dependent.Sum
```

Comment:
Let's add explicit import lists Data.Dependent.Sum as that module has a somewhat bumpy history and I'd like to have clarity about what exactly we depend on. In general *I* prefer explicit import lists, but many others don't.

Reply:
Done, thanks